README.rst

```diff
@@ -78,3 +78,39 @@ Repositories
 .. _`clear-containers-image`: https://download.clearlinux.org/current/
 
 .. _`clear-containers-selinux`: https://github.com/clearcontainers/proxy/tree/master/selinux
+
+Local Building
+==============
+
+Any user can build their own Clear Containers packages using the source files provided in this
+repository, either using the OBS platform or in their own machines using the osc (Open Build Service command line tool).
+
+A Dockerfile is provided to build a container that has all the tools needed to perform local builds.
+
+.. code-block:: shell
+
+   # cd into this repository
+   cd < this repo location>
```

Comment:
Can you change all the commands to have a `$ ` prefix? Admittedly, this means it's slightly awkward to copy-and-paste, but that's the convention we always use to show commands the user runs.